modules/efk-logging-fluentd-json.adoc

```diff
@@ -0,0 +1,73 @@
+// Module included in the following assemblies:
+//
+// * logging/efk-logging-fluentd.adoc
+
+[id="efk-logging-fluentd-json-{context}"]
+= Configuring Fluentd JSON parsing
+
+You can configure Fluentd to inspect each log message to determine if the message is in *JSON* format and merge
+the message into the JSON payload document posted to Elasticsearch. This feature is disabled by default.
+
+You can enable or disable this feature by editing the `MERGE_JSON_LOG` environment variable in the *fluentd* daemonset.
+
+[IMPORTANT]
+====
+Enabling this feature comes with risks, including:
+
+* Possible log loss due to Elasticsearch rejecting documents due to inconsistent type mappings.
+* Potential buffer storage leak caused by rejected message cycling.
+* Overwrite of data for field with same names.
+
+The features in this topic should be used by only experienced Fluentd and Elasticsearch users.
+====
+
+.Prerequisite
+
+* Set cluster logging to the unmanaged state. In managed state, the Cluster Logging Operator reverts changes made to the `fluentd` configuration map.
```

Comment:
Do we have instructions somewhere about how to do this? e.g.
```bash
oc -n openshift-logging patch clusterlogging instance --type=json --patch '[
        {"op":"replace","path":"/spec/managementState","value":"Unmanaged"}]'
```

Reply:
@richm We have instructions to modify the CL CR to set unmanaged state. For example:
http://file.rdu.redhat.com/mburke/logging-config-subdir/logging/config/efk-logging-management.html

Is the patching method _better_?

Reply:
No, not better, just another way to do the same thing